k8s/crds.yaml

```diff
@@ -1274,6 +1274,10 @@ spec:
     subresources:
       status: {}
   - additionalPrinterColumns:
+    - description: The backup target name
+      jsonPath: .spec.backupTargetName
+      name: BackupTarget
+      type: string
```

Comment:
Need generate the crds.yaml by k8s/generate_code.sh rather than update it by manual

Reply:
Regenerated by ./k8s/generate_code.sh. Thanks.